201-api-management-logs-to-moesif-using-eventhub-webapp/azuredeploy.json

```diff
@@ -76,15 +62,16 @@
         "moesifSessionToken": "optional",
         "moesifApiVersion": "v1",
         "apiManagementLoggerId": "moesif-log-to-event-hub",
+        "azureWebsitesDomain": ".azurewebsites.net",
```

Comment:
you cannot hardocde the endpoint, it needs to be valid in other clouds, e.g.

    "azureWebsitesDomainLookup":{
      "AzureCloud": "azurewebsites.net",
      "AzureUSGovernment": "azurewebsites.us"
    },
    "azureWebsitesDomain": "[variables('azureWebsitesDomainLookup')[environment().name]]"

Reply:
This was very helpful, and I am incorporating these changes. Will submit shortly. For now, I have still set AzureCloud as only environment in metadata.json, but above guidance makes it tempting to add in US Govt cloud for future PR.

Reply:
Unless we know there is a specific feature not supported in USGov, we should not block it.  If it hasn't been tested there remove the environments property from metadata.json so we can run the test.

Reply:
Fixed. Added GovCloud and it passes tests. All tests are green right now.